packages/core/contracts/zodiac/OptimisticGovernor.sol

```diff
@@ -96,14 +100,10 @@ contract OptimisticGovernor is Module, Lockable {
             uint64 _liveness
         ) = abi.decode(initializeParams, (address, address, address, uint256, string, bytes32, uint64));
         finder = FinderInterface(_finder);
-        require(_getCollateralWhitelist().isOnWhitelist(address(_collateral)), "bond token not supported");
```

Comment:
In future FYI, have all require statements come before state changes. Two reasons: 
- gas savings on reverts: less gas used paid by caller by exiting earlier.
- reentrancy,  not an issue here, but if you can re-enter with the same state as the previous state, you can do evil things